packages/scaffold-ui/src/partials/w3m-connect-recent-widget/index.ts

```diff
@@ -84,6 +84,26 @@ export class W3mConnectRecentWidget extends LitElement {
 
     ConnectorController.selectWalletConnector(wallet)
   }
+
+  private hasNoWalletConnector(wallet: WcWallet) {
+    return !this.connectors.some(
+      connector => connector.id === wallet.id || connector.name === wallet.name
+    )
+  }
```

Comment:
it is usually a good practice to NOT use negated naming of functions/methods. 
So in this case it's preferred to do 

```
  private hasWalletConnector(wallet: WcWallet) {
    return this.connectors.some(
      connector => connector.id === wallet.id || connector.name === wallet.name
    )
  }
  ```
  
  and do
  
  ```
  .filter(wallet => !this.hasWalletConnector(wallet))
  ```